source/install/desktop.rst

```diff
@@ -85,6 +85,12 @@ Unofficial, community-driven .deb packages are available.
 
 3. To run Mattermost, open **Dash** (located at top left corner) and input ``mattermost``, then click the Mattermost icon.
 
+Arch Linux-based systems
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+`Use the following online guide from the Arch Linux wiki<https://wiki.archlinux.org/index.php/Mattermost>`_.
```

Comment:
Can you change this to:

``To install the desktop client on Arch Linux, see the `Mattermost page <https://wiki.archlinux.org/index.php/Mattermost>`_ on the Arch Linux wiki.``